src/js/base/runtime.js

```diff
@@ -7,6 +7,7 @@ define("pyret-base/js/runtime",
    "pyret-base/js/secure-loader",
    "seedrandom"],
 function (Namespace, jsnums, codePoint, util, exnStackParser, loader, seedrandom) {
+  Error.stackTraceLimit = Infinity;
```

Comment:
Had to add this for some of the tests with larger stack traces. If you think there's a better place for it let me know